apps/web/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM calcom/cal.com
+
+WORKDIR /calcom
+
+COPY ./.env.example ./.env.example
+COPY ./.env.appStore.example ./.env.appStore.example
+
+RUN cp ./.env.example ./.env
+RUN cp ./.env.appStore.example ./.env.appStore
+
+# Create NEXTAUTH_SECRET on .env
+RUN echo NEXTAUTH_SECRET=\"$(openssl rand -base64 32)\" >> ./.env
+
+# Create CALENDSO_ENCRYPTION_KEY on .env
+RUN echo CALENDSO_ENCRYPTION_KEY=\"$(openssl rand -base64 32)\" >> ./.env
```

Comment:
Isn't this supposed to be ? `openssl rand -base64 24`

I saw the docs mentioning this command. Not sure if I'm correct.

Reply:
Fixed; thank you for the feedback!